src/supplemental/mqtt/mqtt_msg.c

```diff
@@ -674,4 +674,65 @@ nni_mqtt_msg_set_aio(nni_msg *msg, nni_aio *aio)
 {
 	nni_mqtt_proto_data *proto_data = nni_msg_get_proto_data(msg);
 	proto_data->aio = aio;
+}
+
+conn_param *
+nni_mqtt_msg_set_conn_param(nni_msg *msg)
```

Comment:
Such a ridiculous function name... I thought it was a get operation

Reply:
should be changed